requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyinstaller>=4.10
+black>=22.1
+mypy>=0.941
```

Comment:
While I don't mind if individual developers use `black` or `mypy` I do not think they should be considered dependencies for the project.

Reply:
having some sort of formatting standard for what you want to "enforce" ensures that others commit code that is visually consistent. I'm fine if you don't want to use mypy but using something for formatting is a good idea. Up to you though.